hack/.ci/manifests/cluster/nodeconfig-pd.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: scylla.scylladb.com/v1alpha1
+kind: NodeConfig
+metadata:
+  name: cluster
+spec:
+  localDiskSetup:
+    loopDevices:
+      - name: persistent-volumes
+        size: 50Gi
```

Comment:
is this because its not actually used? otherwise I was hoping for less (the local SSD are cheaper in the end if you actually use the space they give you)

Reply:
i think we use like 40GiB disk in CI or something like that

Reply:
Changed it to accept path to nodeconfig, this will allow to specify nodeconfig spec alongside kubernetes cluster spec.